test/Passes/fold-pack-and-constant.mlir

```diff
@@ -13,21 +13,6 @@ func.func @expect_to_fold_cst() ->  tensor<8x2x1x1x32x32xi64> {
 
 // -----
 
-func.func @expect_not_to_fold_cst(%dest : tensor<8x2x1x1x32x32xi64>) -> tensor<8x2x1x1x32x32xi64> {
```

Comment:
We did not fold a splat tensor if the destination was not a tensor.empty; this is because it is unclear if we want to make a constant "writable" after bufferization by copying it into a new buffer, upstream folds in all cases: https://github.com/llvm/llvm-project/commit/d0da3d8393939790bb1a6b3b5a36daeeef000c9b

Reply:
Weird, to me this example should always constant pack... There's no empty here to argue about... Well, now it does.